build.js

```diff
@@ -1,7 +1,10 @@
-#!/usr/bin/env node
+#! /usr/bin/env node
 
 'use strict'
 
+// BUILD.JS: This file is responsible for building the HTML site and serving it
+// as a static server.
```

Comment:
"This file is responsible for building static HTML pages and a server for local development."?